test/unit/associations/belongs-to-many.test.js

```diff
@@ -11,7 +11,7 @@ const BelongsTo = require(__dirname + '/../../../lib/associations/belongs-to');
 const HasMany = require(__dirname + '/../../../lib/associations/has-many');
 const HasOne = require(__dirname + '/../../../lib/associations/has-one');
 const current = Support.sequelize;
-const Promise = current.Promise;
+var Promise = current.Promise;
```

Comment:
here we are: https://github.com/jshint/jshint/issues/1747#issuecomment-52574352

I added a jshint exception to the file